src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -212,6 +214,7 @@ def create_project_uptime_subscription(
             headers=headers,
             body=body,
             trace_sampling=trace_sampling,
+            uptime_status=uptime_status,
```

Comment:
This is actually a write change, but we are never calling this function with the uptime_status, so this is actually a no-op change and is only required for tests to pass in the `test_results_consumer` tests